### DISCUSSION TITLE
I want to get my hands dirty, you too?

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#997] [Ideas]
OK, let's pose this question as an idea ...
I'd like to open this thread (here or better on Discord?) to discuss good approaches on how to get your hands dirty and gain real-life project experience. Main obstacles are 

1. just not yet enough skills
2. insufficient experience in projects and collaboration methods. This is a hen egg problem. I can't get into a project without real-life experience and I can't get real-life experience without getting into a project.
3. insufficient experience in finding and picking the right projects to your skill level.

Point one has just one solution and this is learning. 
Points two and three are worth discussing. I would love to hear from people who had made this journey and can comment on what works and what not and what would be considered best practice.

Your thoughts on this? ...

**Comment by cromewar** [comment]
Hello @n4n0b1t3 
I think you described a feeling most of as had more than once and the best solution I found so far was: Hackathons...
Most of the popular and fast growing blockchain startups are constantly creating hackathons, on which you can learn new tech, work on more advanced projects, meet people and also participate for high prizes.

So I definitely recommend you to join one and start building!

**Reply by n4n0b1t3** [reply]
great advice, thank you!

**Comment by alexkalen** [comment]
Is there a discord server for this course? If so I would be glad to join.
